superbench/executor/executor.py

```diff
@@ -122,21 +123,38 @@ def __exec_benchmark(self, context, log_suffix):
             logger.error('Executor failed in %s.', log_suffix)
         return None
 
+    def __get_benchmark_dir(self, benchmark_name):
+        """Get output directory for benchmark's current rank.
+
+        Args:
+            benchmark_name (str): Benchmark name.
+        """
+        benchmark_output_dir = self._output_path / 'benchmarks' / benchmark_name
+        for rank_env in ['PROC_RANK', 'LOCAL_RANK']:
```

Comment:
Can we say rank value match the gpu id for both micro and model benchmark? In other words, how can we match the log to hardware?

Reply:
it's decided by benchmark and its config, e.g., you can use gpu_id=rank/2, or gpu_id=rank*2, etc. for pytorch ddp, gpu_id=rank.